core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review-modal.component.ts

```diff
@@ -199,9 +198,15 @@ export class TranslationSuggestionReviewModalComponent implements OnInit {
         .registerContributorDashboardViewSuggestionForReview('Translation');
       this.heading = 'Review Translation Contributions';
     }
+    const suggestionIdsList = Object.keys(this.suggestionIdToContribution);
+    const clickedSuggestionIndex = suggestionIdsList.indexOf(
+      this.activeSuggestionId);
+    this.skippedContributionIds.push(...suggestionIdsList.slice(
+      0, clickedSuggestionIndex));
     delete this.suggestionIdToContribution[this.initialSuggestionId];
-    this.remainingContributionIds = Object.keys(
-      this.suggestionIdToContribution);
+    this.remainingContributionIds = Object.keys(this.suggestionIdToContribution)
+      .filter(contributionId => !this.skippedContributionIds.includes(
+        contributionId));
```

Comment:
Isn't this equivalent to `suggestionIdsList.slice(clickedSuggestionIndex + 1, suggestionIdsList.length)`?

Reply:
Yup, thanks for catching this. Done.